Description: Minor cuda9 issues

--- psrdada-0+git20180219.orig/gmrt/src/mac_device.hpp
+++ psrdada-0+git20180219/gmrt/src/mac_device.hpp
@@ -1,3 +1,4 @@
+#include <cuda_fp16.h>
 
 #pragma once
 
--- psrdada-0+git20180219.orig/mopsr/src/mopsr_cuda.cu
+++ psrdada-0+git20180219/mopsr/src/mopsr_cuda.cu
@@ -1521,7 +1521,7 @@ void mopsr_tile_beams_precomp (cudaStrea
     fprintf (stderr, "WARNING: ndat not divisible by %d\n", ndat_per_block);
 
   size_t sdata_bytes = (nbeam_block * (nthread/WARP_SIZE) * sizeof(float) * 2) +
-                       (nbeam_block * nant * sizeof (complex float));
+                       (nbeam_block * nant * sizeof ( cuFloatComplex ) );
 
 #ifdef _GDEBUG
   fprintf (stderr, "bytes=%lu ndat=%lu blocks=(%u,%u,%u) threads=%u shm=%ld\n", bytes, ndat, blocks.x, blocks.y, blocks.z, nthread, sdata_bytes);
--- psrdada-0+git20180219.orig/mopsr/src/mopsr_delays.h
+++ psrdada-0+git20180219/mopsr/src/mopsr_delays.h
@@ -115,9 +115,6 @@ static inline float sinc(float x)
 void sinc_filter(complex16* series_in, complex16* series_out,
                  float * filter, int size, int ntaps, float delay);
 
-void sinc_filter_float (complex float * series_in, complex float * series_out,
-                        float * filter, int size, int ntaps, float delay);
-
 mopsr_module_t * read_modules_file (const char* fname, int * nmod);
 
 mopsr_bay_t * read_bays_file (const char* fname, int *nbay);
