Description: another pragma error

--- psrdada-0.0~git20180606.754a618.orig/bpsr/swinproc/dedisp/dedisperse_all.C
+++ psrdada-0.0~git20180606.754a618/bpsr/swinproc/dedisp/dedisperse_all.C
@@ -152,8 +152,9 @@ void do_dedispersion(unsigned short int
 	  if (killdata[k]==1){
 	    idelay = DM_shift(DMtrial,k-start_chan,tsamp,fch1_subband,foff);
 	    int stride = k*ntoload+idelay;
+        int j;
 #pragma omp parallel for private(j)
-	    for (int j=0;j<ntodedisp/4;j++){
+	    for (j=0;j<ntodedisp/4;j++){
 		casted_times[j]+=*((LONG64BIT*) (unpackeddata+(j*4+stride)));
 	    }
 	  } // killdata
